app/Http/Controllers/AttendanceController.php

```diff
@@ -53,18 +53,18 @@ public function store(Request $request)
         unset($request['includeName']);
 
         // Variables for comparison below
-        $today = date('Y-m-d');
+        $date = $request->has('created_at') ? $request->input('created_at') : date('Y-m-d');
         $gtid = $request->input('gtid');
 
         try {
-            $attTodayQ = Attendance::where($request->all())->whereDate('created_at', $today);
-            $attTodayCount = $attTodayQ->count();
-            if ($attTodayCount > 0) {
-                Log::debug(get_class().": Found a swipe today ($today) for $gtid - ignoring.");
-                $att = $attTodayQ->first();
+            $attExistingQ = Attendance::where($request->all())->whereDate('created_at', $date);
```

Comment:
Have you tested how this behaves when you specify a `created_at` in the request?

Reply:
Yes, it works with and without `created_at`.